app/interfaces/utils/IServer.ts

```diff
@@ -0,0 +1,5 @@
+import { Request as IReq } from 'restify';
+
+export default interface IRequest extends IReq {
+  client_id: string;
+}
```

Comment:
Need to add linting configuration for eof with newline.

Reply:
The linting stuff is done by the CI. If there will be linting errors the build will fail.